assets/js/src/delete-users.js

```diff
@@ -6,6 +6,25 @@
  */
 
 /*global jQuery, BulkWP*/
+jQuery( document ).ready( function () {
+    var reassignSelectBoxes = jQuery(".reassign-user");
+    var contentDeleteRadios = jQuery(".delete-content");
+
+    reassignSelectBoxes.each( function() {
+        jQuery(this).attr('disabled', 'true');
+    });
+
+    contentDeleteRadios.change( function () {
+        var reassignSelectBox = jQuery(this).parents('tr').find('.reassign-user');
+        if( "true" === jQuery(this).val() ){
+            reassignSelectBox.removeAttr('disabled');
+		} else {
```

Comment:
I am seeing some alignment issues. Check the whitespace

Reply:
Fixed.